api/models/record.js

```diff
@@ -58,6 +58,36 @@ const Record = BaseModel.extend({
       return this.related('trial').url;
     }
   },
+  trialsPerRegistry: function () {
+  return bookshelf.knex
+    .select(
+      bookshelf.knex.raw('primary_register as registry'),
+      bookshelf.knex.raw('count(primary_register)::int')
+    )
+    .from('trialrecords')
+    .groupBy('primary_register')
+    .orderBy('primary_register')
+    .then((rows) => {
+      return rows;
+    });
+  },
+  lastRegistryUpdate: function () {
+  return bookshelf.knex
+    .select(
+      bookshelf.knex.raw('source_id as id'),
+      bookshelf.knex.raw('sources.name as name'),
+      bookshelf.knex.raw('max(trialrecords.updated_at) as updatedate')
+    )
+    .from('trialrecords')
+    .leftJoin('sources', 'source_id', 'sources.id')
+    .groupBy('source_id', 'sources.name')
+    .then((rows) => {
+      return rows.map((item) => {
+        item.updatedate = (item.updatedate !== null) ? item.updatedate : ''
```

Comment:
How can `updatedate` ever be null? All `trialrecords` should have `created_at` and `updated_at` values. If they don't, there's some error in the data.

Also, the name `updatedate` is a bit too generic. It sounds like the same as `updated_at`, but it isn't. Maybe change it to `latest_update_date` or something similar? Make sure you keep the pattern of using underscores in the column names.